components/Sponsor_Faq/FAQ.module.css

```diff
@@ -0,0 +1,214 @@
+.wrapper{
+    color: white;
+    width: 100%;
+    min-height: 100vh;
+    position: relative;
+}
+
+
+.backgrounds{               /* parent div for the background layer */
+    overflow: hidden;
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    z-index: -1;
+}
+.SponsorBack{
+    width: 100%;
+    height: 100%;
+    background-color: #181414;
+}
+.FaqBack{
+    height: 100%;
+    width: 200%;
+    position: absolute;
+    top: 15%;
+    left: -50%;
+    border-top-left-radius: 50%;
+    border-top-right-radius: 50%;
+    background: -moz-linear-gradient(40deg, rgba(255,192,137,1) 10%, rgba(248,207,171,1) 45%, rgba(194,139,255,1) 76%, rgba(140,88,252,1) 100%);
+    background: -webkit-linear-gradient(40deg, rgba(255,192,137,1) 10%, rgba(248,207,171,1) 45%, rgba(194,139,255,1) 76%, rgba(140,88,252,1) 100%);
+    background: linear-gradient(40deg, rgba(255,192,137,1) 10%, rgba(248,207,171,1) 45%, rgba(194,139,255,1) 76%, rgba(140,88,252,1) 100%);
+    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr="#ffc089",endColorstr="#8c58fc",GradientType=1);
+}
+/* Content */
+.sponsor{
+    height: 20%;
+    font-size: 120%;
+    font-family: var(--coming);
+    font-weight: 500;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+.sponsor p{
+    margin-top: 2em;
+}
+.ButtonWrapper{
+    font-size: 0.7em;
+}
+.ButtonWrapper button{
+    margin-top: 0.3em;
+}
+
+.contacts{
+    height: 80%; 
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-top: 2em;
+}
+.faq{
+    padding-top: 10%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center; 
+    text-align: center;
+}
+.faq span{
+    font-size: 6.5em;
+    font-family: var(--title);
+    text-shadow: -1px -1px 0 #000000, 1px -1px 0 #000000,
+                 -1px 1px 0 #000000, 1px 1px 0 #000000;
+    color: #d7b6fa;   
+}
+.subtitle{
+    color: var(--black);
+    font-family: var(--coming);
+    font-size: 2em;    
+    font-weight: 500;
+    text-align: center;
+    margin: 1rem 0 1rem 0;
+}
+.faq button{        /* Contrast color to default Button accessory */
+    color: rgb(0, 0, 0);
+    background-color: rgb(255, 255, 255);
+    box-shadow: 0.5em 0.5em rgb(0, 0, 0);
+    border: 1px rgb(0, 0, 0) solid;
+    margin-top: 0.5em;
+}
+.faq button:hover{
+    box-shadow: 0.25em 0.25em rgb(0, 0, 0);
+    transform: translate(0.25em, 0.25em);
+}
+
+.chat{
+    margin-top: 3rem;  
+    padding: .95rem; 
+    font-family: var(--coming);
+    font-size: larger;
+}
+.chatW{
```

Comment:
- Reduce the width of the white chat bubble as given in the figma file

Reply:
Made the changes required